### ISSUE TITLE
Documentation introduction section TBD

### ISSUE THREAD

**Issue by Faizan-Alam-1** [AtsushiSakai/PythonRobotics#841]
Could I write introduction and subparts of the documentation ?

https://atsushisakai.github.io/PythonRobotics/modules/introduction.html

@AtsushiSakai

**Comment by AtsushiSakai** [comment]
Yeah. PR is welcome. But, please avoid too big PR. Step by step PR is better for review.

**Comment by datatales-with-pankaj** [comment]
Hi, I wanted to contribute and write the introduction and related subparts of the documentation for the OSS project: https://atsushisakai.github.io/PythonRobotics/modules/introduction.html

P.S. @AtsushiSakai , hints are always welcome for any reference material you have pre-decided on in any case, along with any specific instructions for writing the documentation?
